controllers/local_db_statefulset.go

```diff
@@ -211,3 +215,11 @@ func (r *BackstageReconciler) patchLocalDbStatefulSetObj(statefulSet *appsv1.Sta
 
 	return nil
 }
+
+func (r *BackstageReconciler) setDefaultStatefulSetImage(statefulSet *appsv1.StatefulSet) {
+	for i, c := range statefulSet.Spec.Template.Spec.Containers {
```

Comment:
Same point for using the `visitContainers` function instead. And also, to cover the case where the StatefulSet Spec might contain Init Containers for example.

Reply:
Done